graviti/exception.py

```diff
@@ -74,6 +77,7 @@ def __init__(
 
     def __init_subclass__(cls) -> None:
         cls._INDENT = " " * len(cls.__name__)
+        RESPONSE_ERROR_DISTRIBUTOR[cls.MESSAGE_CODE] = cls
```

Comment:
I think use decorator to register the errors into distributor is easier to understand